test/features/odi-certificates.feature

```diff
@@ -1,30 +1,29 @@
-@smoke
 Feature: ODI Certificates
 
-    Scenario: DQL-67 Add Salsa organisation
+    Scenario: Create a new test organisation for subsequent tests
```

Comment:
This is a test, but also a background activity to ensure that other tests pass if they get out of order.
https://sukesh15.gitbooks.io/cucumber-jvm-test-framework-/content/cucumber_-_more_details/understanding_background_in_cucumber.html

Background:
  Given "Admin" as the persona
  When I log  in
  Then ensure organsation "Test Organisation" exists

Reply:
Background runs on each scenario, therefore the first scenario fails when checking if test organisation exists. I have added Background to the feature, but omitted the check for test organisation.

Reply:
nice, it might be a good step to look into the rest api for doing the check, then you won't need to go through so many pages

https://data.qld.gov.au/api/3/action/organization_list
https://data.qld.gov.au/api/3/action/organization_show?id=aboriginal-and-torres-strait-islander-partnerships&include_users=false